server/app/lib/actions/fusor/deployment/rhev/import_template.rb

```diff
@@ -62,7 +62,7 @@ def run
           def run_command(cmd)
             ::Fusor.log.info "Running: #{cmd}"
             status, output = Utils::Fusor::CommandUtils.run_command(cmd)
-            ::Fusor.log.debug "Status: #{status}, output: #{output}"
+            ::Fusor.log.info "Status: #{status}, output: #{output}"
             return status, output
```

Comment:
Not sure this is useful at INFO. We really only care if it failed and that is handled in the run method when we fail the job:

```
status, output = run_command(cmd)
if status != 0
  fail _("Unable to import template: Status: %{status}. Output: %{output}") % { :status => status, :output => output }
end
```